### ISSUE TITLE
"use the same prompt for each image in a batch" doesn't work as intended

### ISSUE THREAD

**Issue by diregoblin** [Klokinator/UnivAICharGen#4]
Thanks for the prompt collections! I've played with them a bit, and it seems when I uncheck the option in question in the WebUI, it only works for the 1st batch. 

I ran batches of four, and in the 1st batch images 1,2,3 had different prompts; image 1.4 had a same one as 1.3. In the 2nd batch image 2.1 was similar to 1.4 and 1.3;  2.2, 2.3, 2.4, 3.1 had a shared prompt; 3.2, 3.3, 3.4 shared a prompt as well. From batch 4 onwards the prompt was shared by the entire batch.

**Comment by Klokinator** [comment]
Yup. Batches don't work in Umi AI properly. That's why we don't use 'em. Unless you know python/how to fix it, it's unlikely to get fixed. Both of my python guys said it's an issue with WebUI's limitations.

**Comment by Klokinator** [comment]
Correction: Batches don't work with the negative prompts in Wildcards, or certain wildcard parameters. This isn't limited to Umi. It's in the other wildcard systems too.

**Comment by diregoblin** [comment]
Interesting. Is there a more technical description of the nature of this
problem anywhere? I know some stuff about python parallelism, may take a
look at it.

Also, I was only using the wildcards for the positive prompt, what are
these "certain wildcard parameters"?

Cheers

сб, 12 нояб. 2022 г. в 12:06, Klokinator ***@***.***>:

**Comment by Klokinator** [comment]
I'm not a coder, so I can't explain it well. If you'd like, you can jump on the discord and we can talk about it there! I have my other two coders who might be able to help explain.

https://discord.gg/9K7j7DTfG2

**Comment by Klokinator** [comment]
I do believe one of my coder guys solved the issue! Try again?

https://github.com/Klokinator/UnivAICharGen/pull/9
